src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,17 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        if (UserService.findByEmail(email) != null) {
+            return UserService.findByEmail(email).getEmail().equals(password);
```

Comment:
you call `.findByEmail(email)` 2 times - let's better save its result into local variable and reuse it